terms/abs/abs.md

```diff
@@ -0,0 +1,51 @@
+## Syntax
+```cs
+Math.Abs(value)
+```
+Math.Abs() is a static method of the Math class. It takes a single parameter of type int, long, float, double, or decimal. It returns the absolute value of the number.
```

Comment:
This should be the introductory sentence of the entry, before the syntax header.